### PR TITLE
Osunick splitscreen

### PR DESCRIPTION
**Pull request by osunick**
bunch of changes to move unit and speed in splitscreen mode.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ android {
         applicationId "app.candash.cluster"
         minSdkVersion 24
         targetSdkVersion 31
-        versionCode 50
```
Comment (nmullaney) [inline android/app/build.gradle:25]: versionCode's usually only increase.  Also, it's good practice to change the versionName at the same time.

File: android/app/src/main/java/app/candash/cluster/DashFragment.kt [hunk 2]
```diff
@@ -426,15 +430,30 @@ class DashFragment : Fragment() {
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R && view.windowToken != null) {
                 // only needed for Android 11+
                 if (isSplit) {
+/*                    val params = binding.speed.layoutParams as ConstraintLayout.LayoutParams
```
Comment (nmullaney) [inline android/app/src/main/java/app/candash/cluster/DashFragment.kt:433]: Should this be deleted?

**Review by nmullaney — APPROVED** [review]
Looks good, but the versionCode change might be a mistake?

**Comment by osunick** [comment]
Oh, because it's commented out? Yeah, probably.

**Pull request merged**